ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomInputField.js

```diff
@@ -24,7 +25,8 @@ const CustomInputField = (props) => {
         error={props.rawErrors?.length > 0}
         onChange={e => props?.onChange(e.target.value)}
         label={`${prettifiedName}`}
-        InputProps={{ style : { padding : "2px 0px 3px 0px", backgroundColor : "rgba(255, 255, 255, 0.4)" },
+        InputProps={{
+          style : { padding : "2px 0px 3px 0px", backgroundColor : "rgba(255, 255, 255, 0.4)" },
```

Comment:
Thanks, @Harshit-Dandriyal. 

It's better that the padding is relative, not absolute, so that it is responsive. 

If not relative, then let's at least use `rem` instead of `px`.

Reply:
The fields are centered and the screenshot does look nice. 👍